src/__test__/NotFound.test.js

```diff
@@ -0,0 +1,30 @@
+import '@testing-library/jest-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import store from '../redux/configureStore';
+import NotFound from '../component/NotFound';
+
+describe('NotFound component testing', () => {
+  test('Does the NotFoud component match snapshot', () => {
+    const notFound = render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <NotFound />
+        </Provider>
+      </BrowserRouter>,
+    );
+    expect(notFound).toMatchSnapshot();
```

Comment:
> When I run `npm run test` -> `This test is failed`, to fix it try to use asFragment

- [OPTIONAL] To Check if DOM loads as usual before dynamic content loads, It would be better to use asFragment.


> For Example 💡
```js
    const { asFragment } = render(
      <BrowserRouter>
        <Provider store={store}>
          <NotFound />
        </Provider>
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
```